arches_lingo/src/arches_lingo/components/generic/NonLocalizedStringEditor.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+import { ref, watch } from "vue";
+import InputText from "primevue/inputtext";
+
+const props = defineProps<{ value: string }>();
+const updateableValue = ref<string>(props.value as string);
+const emit = defineEmits(["update"]);
+watch(props, (newValue) => {
```

Comment:
styling nit: I'd expect newlines between props, declared consts, emits, and `watch`es . However I won't block on this til we flesh out the styleguide to include it 😄

Reply:
better now than later!